### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by hilch**
Add hilch/micropython-switchbot-thermometer-hygrometer

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ Other places you can look for MicroPython Libraries:
 * [ESP8266-Home-Assistant-Smart-Socket](https://github.com/AnthonyKNorman/ESP8266-Home-Assistant-Smart-Socket) - This MicroPython project is to hack a Hyleton313 cheap WiFi smart socket.
 * [ESP8266-Home-Assistant-RGB-Bulb](https://github.com/AnthonyKNorman/ESP8266-Home-Assistant-RGB-Bulb) - This MicroPython project is to hack a TYWE3S board in a cheap WiFi RGB Bulb.
 * [uPyIoT](https://github.com/lemariva/uPyIoT) - Connect an M5Stack ATOM running MicroPython to the Google Cloud Platform (GCP) to collect air-quality variables obtained from reading sensors.
+* [https://github.com/hilch/micropython-switchbot-thermometer-hygrometer] - read SwitchBot Thermometer/Hygrometer via Bluetooth
```
Comment (andypiper) [inline readme.md:192]: The URL needs to be in round brackets () and there should be a title for the link in square brackets [] - please update 🙂

**Review by andypiper — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (hilch) [reply readme.md:192]: argh, sorry :-)

```suggestion
* [micropython-switchbot-thermometer-hygrometer](https://github.com/hilch/micropython-switchbot-thermometer-hygrometer) - read SwitchBot Thermometer/Hygrometer via Bluetooth
```

**Comment by mcauser** [comment]
Thanks @hilch 
Reminds me of the [Xiaomi Mijia Temperature and Humidity Monitor](https://mi-store.com.au/products/xiaomi-temperature-and-humidity-monitor-2)

**Comment by hilch** [comment]
@mcauser : you're welcome. Absolut, that is looking similar. But my ones don't have this smiley display. So I suppose it doesn't contain the same electronics.

**Pull request merged**